src/components/PageWrapper/index.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components'
+
+export const PageWrapper = styled.div`
+  max-width: 856px;
+  width: 100%;
+  margin: 0 auto;
+`
```

Comment:
This is style doesn't belong to components folder...I would suggest moving it to some common styles file inside pages folder or to common styles folder...

Reply:
I also agree for a separate UI folder for components. What about moving to https://mui.com/ or https://flowbite.com/ instead of building a new design system.

Reply:
Moved it to pages folder